### ISSUE TITLE
image has not been displayed correctly in background-image property

### ISSUE THREAD

**Issue by TabrisK** [rodneyrehm/viewport-units-buggyfill#97]
I am using react-css-module to make a SPA.Everything is ok when I use background-image in other places, but not correctly at here.The code and web dev tool's screenshot is as follows.
./promotion.scss:
<img width="513" alt="1" src="https://user-images.githubusercontent.com/12509010/38194000-9eabd964-36a7-11e8-8e0e-a9bfdacce425.png">

./promotion.js:
<img width="498" alt="2" src="https://user-images.githubusercontent.com/12509010/38194002-9f0fb56a-36a7-11e8-8305-eaae10fd0f67.png">

screenshot of style block's in web dev tool:
<img width="322" alt="3" src="https://user-images.githubusercontent.com/12509010/38194003-9f3c19de-36a7-11e8-8102-9e2390bd90f1.png">

error message in console:
<img width="1011" alt="4" src="https://user-images.githubusercontent.com/12509010/38194004-9f6865a2-36a7-11e8-8b27-d01e773d7f3e.png">

**Comment by TabrisK** [comment]
I found the different place between the two image data in background-image.

<img width="377" alt="1" src="https://user-images.githubusercontent.com/12509010/38226365-d13c83a4-372b-11e8-91aa-463a702aec44.png">
The above is the error version, and another is the correct version.

**Comment by rodneyrehm** [comment]
yeah, the `4vh` is replaced but really shouldn't. This problem was first reported in #34 and attempted to be fixed in #86 - would you like to pick up that PR and bring it across the finish line?

**Comment by Jurool** [comment]
I also encountered the same problem and I could not find a solution for the time being.
![error](https://user-images.githubusercontent.com/26647750/41643965-55caf410-749f-11e8-9323-b0e8417bb14a.png)
